index.html

```diff
@@ -55,8 +49,29 @@
         </ul>
       </nav>
     </div>
-
-    <main></main>
+    <main>
+      <div class="main-container-Santouka">
+        <div class="santouka-img">
+          <img src="img/ramen3_shio_santouka.png" alt="Ramen from Santouka">
+        </div>
+        <div class="santouka-desc">
+          <h1>Hokkaido Ramen <p>Santouka</p></h1>
+          <h2>Like food parents give their children, we strive and pour our love into our ramen. In keeping with the spirit of Santouka at the time of its founding, we aim to be a ramen restaurant that will be loved forever by people around the world. We will never stop delivering our consistent flavor.</h2>
+        </div>
+      </div>
+      <div class="main-caption">
+        <p>WHICH RAMEN WOULD YOU LIKE?</p>
+      </div>
+      <div class="main-container-Gojiro">
```

Comment:
css is case insensitive, so you should stick to lower case

css dont know the difference of these 3 items.
they are all same to css.

1. TEST_TEXT
2. test_text
3. Test_Text

Reply:
main-container-Gojiro -> main-container-gojiro